pkg/api/types.go

```diff
@@ -659,6 +955,77 @@ type Unit struct {
 	//initialized. If specified, no other probes are executed until
 	//this completes successfully.
 	StartupProbe *Probe `json:"startupProbe,omitempty"`
+
+	// NOT YET SUPPORTED, ADDED FOR CONFORMANCE
+
+	// List of sources to populate environment variables in the container.
+	// The keys defined within a source must be a C_IDENTIFIER. All invalid keys
+	// will be reported as an event when the container is starting. When a key exists in multiple
+	// sources, the value associated with the last source will take precedence.
+	// Values defined by an Env with a duplicate key will take precedence.
+	// Cannot be updated.
+	// +optional
+	EnvFrom []core.EnvFromSource `json:"envFrom,omitempty"`
+	// Compute Resources required by this container.
+	// Cannot be updated.
+	// More info: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
+	// +optional
+	Resources core.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
I think here we can copy over ResourceRequirements and ResourceList, since they are pretty simple. We can import the resource package that they use, since that is not a k8s api package.